picker-starter/src/data/products.ts

```diff
@@ -0,0 +1,723 @@
+    category: 'Hygiene',
+  },
+  {
+    id: 4997166,
+    filename:
+      'https://s3.amazonaws.com/a.storyblok.com/f/160270/1280x896/4da92eedd7/omega-3-oil-capsules.jpeg',
+    category: 'Food',
+  },
+  {
+    id: 4997165,
+    filename:
+      'https://s3.amazonaws.com/a.storyblok.com/f/160270/1280x896/b09879b9ea/storybeer-33cl.jpeg',
+    category: 'Beverages',
+  },
+  {
+    id: 4997164,
+    filename:
+      'https://s3.amazonaws.com/a.storyblok.com/f/160270/1280x896/0810d93216/storybeer-50cl.jpeg',
+    category: 'Beverages',
+  },
+  {
+    id: 4997163,
+    filename:
+      'https://s3.amazonaws.com/a.storyblok.com/f/160270/1280x896/8f32edf711/apple-cider-50cl.jpeg',
+    category: 'Beverages',
+  },
+  {
+    id: 4997162,
+    filename:
+      'https://s3.amazonaws.com/a.storyblok.com/f/160270/1280x896/15b4afdfbf/tank-top.jpeg',
+    category: 'Clothing',
+  },
+  {
+    id: 4997161,
+    filename:
+      'https://s3.amazonaws.com/a.storyblok.com/f/160270/1280x896/4dec0555bc/storyblok-belgian-beer.jpeg',
+    category: 'Beverages',
+  },
+  {
+    id: 4997160,
+    filename:
+      'https://s3.amazonaws.com/a.storyblok.com/f/160270/1280x896/04d2928721/small-sign-board.jpeg',
+    category: undefined,
+  },
+  {
+    id: 4997159,
+    filename:
+      'https://s3.amazonaws.com/a.storyblok.com/f/160270/1280x853/2077aad3d4/gray-sweater.jpeg',
+    category: 'Clothing',
+  },
+]
+
+const getImage = (filename: string) =>
+  `${filename.replace('s3.amazonaws.com/', '')}/m/500x0`
+
+const getNameFromImage = (filename: string): string | undefined => {
+  const imageName = filename.split('/').splice(-1)[0]
+  return imageName?.split('.')?.[0]?.split('-')?.map(capitalizeWord)?.join(' ')
+}
+
+export type MockProduct = ProductItem & {
+  category: string | undefined
+}
+
+export const products: MockProduct[] = tmpAssets
+  .map((mockProduct) => {
+    const { filename, id, description } = mockProduct
+    const name = filename && getNameFromImage(filename)
+
+    return {
+      type: 'product',
+      id: `${id}`,
+      name: name ?? 'Without image',
+      image: filename && getImage(filename),
+      sku: `sku-${id}`,
+      description,
+      category: mockProduct.category,
+    } as MockProduct
+  })
+  .sort(compareName)
+
+export const getProductOptions: OptionsQuery = async () => {
+  const options: Option[] = categoryMockAssets.map<Option>((category) => {
+    return {
+      label: category.name,
+      value: category.name,
+    }
+  })
+
+  const response: MultiOption[] = [
+    {
+      type: 'single',
+      label: 'Category',
+      options: options,
+      defaultValue: undefined,
+      name: 'category',
+    },
+    {
+      type: 'multi',
+      options: options,
+      defaultValue: [],
+      label: 'Categories',
+      name: 'categoryMulti',
+    },
+  ]
+
+  return delayed(randomDelay(), response)
+}
+
+export const queryProducts: ItemQuery = async ({
+  search_term,
+  page,
+  per_page,
+  userOptions,
```

Comment:
inconsistent casing. let's stick to `camelCase'.

Reply:
Fixed [here](https://github.com/storyblok/field-type-examples/pull/80/commits/5be2f8e7fc925e615bffe68ce14c3ce0ebc324a3), @eunjae-lee 🙌